superset/db_engine_specs.py

```diff
@@ -590,6 +590,58 @@ def get_table_names(cls, inspector, schema):
         """Need to disregard the schema for Sqlite"""
         return sorted(inspector.get_table_names())
 
+class DrillEngineSpec(BaseEngineSpec):
+    """Engine spec for Apache Drill"""
+    engine = 'drill'
+
+    @classmethod
+    def extra_table_metadata(cls, database, table_name, schema_name):
+        """Returns engine-specific table metadata"""
+
+        return {}
+
+    @classmethod
+    def select_star(cls, my_db, table_name, engine, schema=None, limit=100,
+                    show_cols=False, indent=True, latest_partition=True,
+                    cols=None):
+        table_name = table_name.replace('@@@', '.')
+
+        fields = '*'
```

Comment:
This method looks largely copy-pastaed from the base class, why not just calling `super` with the altered `table_name`?

Reply:
Fixed